### ISSUE TITLE
Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.

### ISSUE THREAD

**Issue by cristiankmb** [ihmpavel/expo-video-player#4]
I'm trying to play a video following the steps from [expo-video-player](https://github.com/ihmpavel/expo-video-player) however i can't run the component. Always display the same error and i don't idea that how i can fix it. i only test Android Device

You can see on this Snack [example](https://snack.expo.io/B1wqykbQr):

app.json:
`"sdkVersion": "34.0.0",
`
package.json:
```
"expo": "^34.0.3",
"expo-av": "^6.0.0",
"expo-video-player": "^1.2.0",
"react": "16.8.3",
"react-native": "https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz",
```

Error result:

**Invariant Violation: Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.**

Check the render method of `VideoPlayer`.
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:5716:10 in createFiberFromTypeAndProps
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:5744:4 in createFiberFromElement
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:8318:12 in createChild
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:8661:10 in reconcileChildrenArray
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:10994:6 in reconcileChildren
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:11755:4 in updateHostComponent
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:17276:21 in performUnitOfWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:17316:41 in workLoop
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:17417:15 in renderRoot
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:18423:17 in performWorkOnRoot
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:18324:24 in performWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:18285:14 in performSyncWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:18169:19 in requestWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:17969:16 in scheduleWork
- node_modules/react-native/Libraries/Renderer/oss/ReactNativeRenderer-dev.js:6934:17 in enqueueSetState
- node_modules/react/cjs/react.development.js:335:31 in setState
- node_modules/react-navigation/src/createNavigationContainer.js:342:22 in dispatch
- node_modules/react-native-router-flux/src/navigationStore.js:891:31 in dispatch
- node_modules/react-native-router-flux/src/navigationStore.js:908:18 in push
- node_modules/react-native-router-flux/src/navigationStore.js:902:17 in execute
* src/components/newProgram/home/index.js:353:21 in <unknown>
- node_modules/@firebase/database/dist/cjs/src/api/Query.js:244:51 in onceCallback
- node_modules/@firebase/database/dist/cjs/src/core/view/EventRegistration.js:65:26 in <unknown>
- ... 21 more stack frames from framework internals

**Comment by rafmrss** [comment]
I got the exact same problem, any fix for this?

**Comment by cristiankmb** [comment]
@rafmrss Not yet. I need to show a video on landscape, **expo-video-player** was the best option but since his last update i couldn't use it again. Then i had to search another solution, i started to use video from [expo](https://docs.expo.io/versions/latest/sdk/video/), temporarily.

**Comment by rafmrss** [comment]
Nice tip @cristiankmb

**Comment by janrop** [comment]
Does anybody know of an alternative repo wich is compatible with expos sdk 34?

**Comment by cristiankmb** [comment]
@janrop  I think there is any yet, because for Expo SDK 34 is necesary use the new updated component expo-video-player, however you can try use [expo-av] (https://docs.expo.io/versions/latest/sdk/av/) that have video component very useful.

**Comment by ihmpavel** [comment]
Hi, I will update the package and it should fix everything. 

I hope, it should be ready tomorrow.

**Comment by ihmpavel** [comment]
Fixed in version 1.3.0 #5
